src/Simplex/Chat/Store/Messages.hs

```diff
@@ -582,58 +596,53 @@ findDirectChatPreviews_ db User {userId} pagination clq =
     baseParams = (userId, userId, CISRcvNew)
     getPreviews = case clq of
       CLQFilters {favorite = False, unread = False} -> do
-        let q = baseQuery <> " WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used"
+        let q = baseQuery <> " WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used = 1"
             p = baseParams :. Only userId
-        queryWithPagination db q p pagination
+        queryWithPagination q p
       CLQFilters {favorite = True, unread = False} -> do
         let q =
               baseQuery
-                <> [sql|
-                      WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used
-                        AND ct.favorite = 1
-                  |]
+                <> ( " WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used = 1"
+                      <> " AND ct.favorite = 1"
+                   )
             p = baseParams :. Only userId
-        queryWithPagination db q p pagination
+        queryWithPagination q p
       CLQFilters {favorite = False, unread = True} -> do
         let q =
               baseQuery
-                <> [sql|
-                      WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used
-                        AND (ct.unread_chat = 1 OR ChatStats.UnreadCount > 0)
-                  |]
+                <> ( " WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used = 1"
+                      <> " AND (ct.unread_chat = 1 OR ChatStats.UnreadCount > 0)"
+                   )
             p = baseParams :. Only userId
-        queryWithPagination db q p pagination
+        queryWithPagination q p
       CLQFilters {favorite = True, unread = True} -> do
         let q =
               baseQuery
-                <> [sql|
-                      WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used
-                        AND (ct.favorite = 1
-                          OR ct.unread_chat = 1 OR ChatStats.UnreadCount > 0)
-                  |]
+                <> ( " WHERE ct.user_id = ? AND ct.is_user = 0 AND ct.deleted = 0 AND ct.contact_used = 1"
```

Comment:
this and other similar changes don't seem to be necessary, baseQuery <> [sql||] compiles as is